gtars-igd/src/lib.rs

```diff
@@ -3,6 +3,9 @@
 pub mod create;
 pub mod search;
 
+#[cfg(feature = "bloom")]
```

Comment:
Since this feature flag is here... you might not need them in `igdbloom.rs`, right?